### PR TITLE
Add READMEs for each appengine flex project.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: appengine/flexible/RedisCache/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+5.  Create a [Redis Database using Redis Cloud](
+        https://redislabs.com/redis-howto#create-a-database-using-redis-cloud).
+    Be sure to specify `GCE/us-central1` as the Cloud for the lowest latency.
+    Specify a **strong password** as your Redis instance will be publicly
+    accessible.  Note the Endpoint for the database.
+
+4.  Edit [appsettings.json](appsettings.json).  Replace 
+    `your-redis-config-string` with your Redis Endpoint.
+
```
Comment (jsimonweb) [inline appengine/flexible/RedisCache/README.md]: Suggestion: Add example connection with additional instructions:
======   
Here's an example Redis configuration string. Replace [Redis-Endpoint] with your Redis Endpoint URL. Replace [Redis-Password] with your Redis password.

   REDIS_CONFIG: "[Redis-Endpoint],password=[Redis-Password]"

======

File: appengine/flexible/StaticContent/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Static Content sample for Google App Engine Flexible Environment
+
+This sample application demonstrates how to server static content with
```
Comment (jsimonweb) [inline appengine/flexible/StaticContent/README.md]: ~ serve static content

File: appengine/flexible/RedisCache/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+## Run Locally
+
+### Using PowerShell
+```psm1
+PS > dotnet restore
+PS > dotnet run
+```
+
+### Using Visual Studio
+Press F5.
```
Comment (jsimonweb) [inline appengine/flexible/RedisCache/README.md]: 1.  In Solution Explorer, right-click the **RedisCache** project and choose **Set as StartUp Project**

2. Press F5.

File: appengine/flexible/SendGrid/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## Run Locally
+
+### Using PowerShell
+```psm1
+PS > dotnet restore
+PS > dotnet run
+```
+
+### Using Visual Studio
+Press F5.
```
Comment (jsimonweb) [inline appengine/flexible/SendGrid/README.md]: 1.  In Solution Explorer, right-click the **SendGrid** project and choose **Set as StartUp Project**

2. Press F5.

File: appengine/flexible/CloudSql/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    make it easy to deploy to App Engine.  Install them if you are running Visual Studio.
+
+4.  [Create a second generation Google Cloud SQL instance.](
+    https://cloud.google.com/sql/docs/mysql/create-instance).
+
+5.  [Configure access to the Google Cloud SQL instance.](
+    https://cloud.google.com/sql/docs/mysql/instance-access-control)
+
+6.  Edit [appsettings.json](appsettings.json).  Replace `your-cloud-sql-connection-string` with your cloud sql connection string.
+
```
Comment (jsimonweb) [inline appengine/flexible/CloudSql/README.md]: Suggestion: provide example connection string noting which parts of the connection string to replace.

Comment (SurferJeffAtGoogle) [inline appengine/flexible/CloudSql/README.md]: It's really complicated.  I added a link to doc that explains how to construct the connection string.

File: appengine/flexible/CloudSql/README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+## Run Locally
+
+### Using PowerShell
+```psm1
+PS > dotnet restore
+PS > dotnet run
+```
+
+### Using Visual Studio
+Press F5.
```
Comment (SurferJeffAtGoogle) [inline appengine/flexible/CloudSql/README.md]: Done.

File: appengine/flexible/RedisCache/README.md (hunk 3 above)
Comment (SurferJeffAtGoogle) [inline appengine/flexible/RedisCache/README.md]: Done.

File: appengine/flexible/StaticContent/README.md (hunk 2 above)
Comment (SurferJeffAtGoogle) [inline appengine/flexible/StaticContent/README.md]: Done.

File: appengine/flexible/SendGrid/README.md (hunk 4 above)
Comment (SurferJeffAtGoogle) [inline appengine/flexible/SendGrid/README.md]: Done.

**Pull request merged**